salt/_modules/metalk8s_drain.py

```diff
@@ -357,7 +357,7 @@ def run_drain(self, dry_run=False):
         try:
             pods = self.get_pods_for_eviction()
         except DrainException as exc:
-            raise CommandExecutionError(
+            raise CommandExecutionError(  # pylint: disable=raise-missing-from
```

Comment:
Can't the `from` be added instead?

Reply:
We can but not sure it bring any value, you will just get the traceback to see from where it gets raised and it's will likely be on line 
```
        if failures:
            raise DrainException(_message_from_pods_dict(failures))
```
So ....

Reply:
What if we later change the `DrainException` to be raised from another one? IMO, if `pylint` enforces it as a rule, a `raise Something from caught_exception` never hurts.

Reply:
Ok fair